pkg/bkctl/ledger/get_test.go

```diff
@@ -23,6 +23,15 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestGetCmd(t *testing.T) {
+	args := []string{"get", "1"}
```

Comment:
Same comment as above

Reply:
This command is failed. It returns a server error. Because we do not catch the exception, it will return an internal server error.